src/layouts/AdminPage.js

```diff
@@ -104,6 +103,28 @@ export class AdminPage extends Component {
               </Form>
             </Grid.Column>
           </Grid.Row>
+          <Grid.Row>
+            <Grid.Column width={4} textAlign="left">
+              <b>Paths for images</b>
```

Comment:
I don't see this new feature? How do I activate it?

Reply:
Yes that's the issue I intended to solve. There was alredy 2 different ways to manage config in the project: one on config.py files and another using constance. The second is way better, so I moved the data (that used config.py) to constance but I needed a way to set it up.

You have to get the development build and build images so that code is being pulled from this pull request, otherwise, using precompiled images, you are not using this code at all. I'm attaching a preview btw.

![image](https://user-images.githubusercontent.com/1440608/107890615-2b3a3580-6f1a-11eb-9fa2-bf8d3868e0b8.png)

Reply:
Ah, I see what you were trying to change. We currently only use the /data folder. I think we don't have to see or change that for now. LibrePhotos/librephotos/issues/123 wants to extend the user model so that one user can select multiple folders within the data mount and have some options how each folder is scanned.

Reply:
Ah ok, I misunderstood. So on application startup I can set that conf to be /data  and that's it right?

Reply:
Ok, so now we have /data default and removed the selector.